libs/oeth/history/src/components/ChartCard.tsx

```diff
@@ -0,0 +1,104 @@
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+import { Card } from '@origin/shared/components';
+import React, { useLayoutEffect, useRef } from 'react';
+import { useIntl } from 'react-intl';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  LineElement,
+  PointElement,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+
+ChartJS.register(CategoryScale, LinearScale, LineElement, PointElement);
```

Comment:
Maybe move this to `libs/oeth/shared/src/clients/chart.ts` as it should be called once

Reply:
done, moved to `shared/providers/chart`